### PR TITLE
Added on README instructions to installl and config nomos

### LINKED ISSUE #75: Add nomos usage instructions to validate Config Management System changes
## Expected Behavior

in step 6, after we push the changes for the Config Management System we should know if they were successfully applied.

## Actual Behavior

There is no feedback if the Config Management System changes wer successful.

The only way is to go to the Logs Explorer in the GKE project and search the reconciler logs using: 

```
resource.type="k8s_container"
resource.labels.namespace_name="config-management-system"
resource.labels.container_name="reconciler"
```

## Steps to Reproduce the Problem

1. deploy up to step 6 
1. add an invalid change in one of the yaml  files under `root-config-repo`
1. add a valid change in another yaml files under `root-config-repo`.
1. push the change to the repo 
1. There is no clear indication that the update failed but for the fact that you can't see any of the changes in the Kubernetes Engine object browser.

## Suggestion

- Add instructions to install [nomos](https://cloud.google.com/anthos-config-management/docs/how-to/nomos-command) for the same version of the Config Management System in a folder in the path.
- Add instructions to validate the configuration (`nomos vet`) and to check if the update was successful (`nomos status`)

## Specifications

- Version: de00f4fe2540912949a6fc32691c14964211c2d3
- Platform: Ubuntu 20.04

### PR DESCRIPTION
**Pull request by vfigueiredo-cit**
Fixes #75 

> It's a good idea to open an issue first for discussion.

- [ ] Tests pass
- [ ] Appropriate changes to README are included in PR

### CODE REVIEW

File: 6-anthos-install/README.md [hunk 1]
```diff
@@ -291,6 +291,31 @@ Create a secret with your private key in both clusters.
 
     kubectl apply --context=${CTX_2} -f ${HOME}/terraform-example-foundation-app/6-anthos-install/acm-configs/config-management-west.yaml
 
+### Install Nomos
+Nomos will be installed to know if the Config Management System changes were successfully applied.
```
Comment (bharathkkb) [inline 6-anthos-install/README.md:295]: I think we can link to the upstream docs to install nomos (so we don't have to keep install instructions upto date)

File: 6-anthos-install/README.md [hunk 2]
```diff
@@ -291,6 +291,31 @@ Create a secret with your private key in both clusters.
...
+    chmod +x nomos
+    ```
+
+1. Move the command to a location the system searches for binaries.
+    ```
+    sudo mv nomos /usr/local/bin/
+
+1. Validate the configuration.
+    ```
+    nomos --path=${HOME}/bank-of-anthos-repos/root-config-repo vet
```
Comment (bharathkkb) [inline 6-anthos-install/README.md:312]: This path may not always be `${HOME}/bank-of-anthos-repos` right? We should just tell the users to run relative to this repo root.

**Review by bharathkkb — CHANGES_REQUESTED** [review]
Thanks for the PR @vfigueiredo-cit

File: 6-anthos-install/README.md (hunk 2 above)
Comment (bharathkkb) [reply 6-anthos-install/README.md:312]: I realized this was in the bastion and we do ask explicitly to use `${HOME}/bank-of-anthos-repos`.

File: 6-anthos-install/README.md (hunk 1 above)
Comment (vfigueiredo-cit) [reply 6-anthos-install/README.md:295]: It is a good idea.

**Pull request closed without merging**